perception/lidar_centerpoint/launch/centerpoint_vs_centerpoint-tiny/README

```diff
@@ -0,0 +1,138 @@
+# Run lidar_centerpoint and lidar_centerpoint-tiny simultaneously
+
+This tutorial is for showing `centerpoint` and `centerpoint_tiny`models’ results simultaneously, making it easier to visualize and compare the performance.
+
+## Setup Development Environment
+
+Follow the steps in the Source Installation ([link](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/)) in Autoware doc.
+
+If you fail to build autoware environment according to lack of memory, then it is recommended to build autoware sequentially.
+
+Source the ROS 2 Galactic setup script.
+
+```bash
+source /opt/ros/galactic/setup.bash
+```
+
+Build the entire autoware repository.
+
+```bash
+colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release --parallel-workers=1
+```
+
+Or you can use a constrained number of CPU to build only one package.
+
+```bash
+export MAKEFLAGS="-j 4" && MAKE_JOBS=4 colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release --parallel-workers 1 --packages-select PACKAGE_NAME
+```
+
+Source the package.
+
+```bash
+source install/setup.bash
+```
+
+## Data Preparation
+
+### Using rosbag dataset
+
```

Comment:
[imo] Could you  add a statement that developers can also use the sample rosbag provided  [here](https://autowarefoundation.github.io/autoware-documentation/main/tutorials/ad-hoc-simulation/rosbag-replay-simulation/)

Reply:
Added!